src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -2374,6 +2375,26 @@ default List<Emote> getEmotesByName(@Nonnull String name, boolean ignoreCase)
      * @see #shutdown()
      */
     void shutdownNow();
+    
+    /**
+     * Blocks until this JDA instance fully shuts down or the timeout expires. <b>This method does not shut down JDA.</b>
+     * 
+     * <p>If you want to run code after shutting down async, use the {@link ShutdownEvent} in preference to this method.
+     * 
+     * @param timeout
+     *        The timeout to wait for shutdown before returning.
+     *        
+     * @param unit
+     *        {@link TimeUnit Unit} for the timeout value.
+     *        
+     * @throws InterruptedException
+     *          if the current thread is interrupted while waiting.
+     *          
+     * @return true if shutdown completed, false if the timeout expires.
+     * 
+     * @see {@link net.dv8tion.jda.api.events.ShutdownEvent ShutdownEvent}
```

Comment:
```suggestion
     * @param  timeout
     *         The timeout to wait for shutdown before returning.
     *        
     * @param  unit
     *         {@link TimeUnit Unit} for the timeout value.
     *        
     * @throws InterruptedException
     *          if the current thread is interrupted while waiting.
     *          
     * @return true if shutdown completed, false if the timeout expires.
     * 
     * @see    net.dv8tion.jda.api.events.ShutdownEvent ShutdownEvent
```
These should all follow the same indentation, follow the [Structure-Guide](https://github.com/DV8FromTheWorld/JDA/wiki/6%29-JDA-Structure-Guide)!

@see shouldn't use @link (if you tried you saw the see didn't work)